core/templates/components/ck-editor-helpers/ck-editor-4-rte.component.ts

```diff
@@ -307,22 +307,22 @@ export class CkEditor4RteComponent implements AfterViewInit, OnChanges,
     }
 
     /**
-       * Create rules to whitelist all the rich text components and
+       * Create rules to allowlist all the rich text components and
```

Comment:
Again, rather than using allowlist it might make sense to just say `allow`.

Reply:
Done with this suggested change too.